website/docs/getting-started/build-a-sample-app.mdx

```diff
@@ -8,12 +8,28 @@ the boilerplate needed for a basic Yew app or manually set up a small project.
 ## Using a starter template
 
 Install [`cargo-generate`](https://github.com/cargo-generate/cargo-generate) by following their installation instructions
-then run the following command:
+then take the following steps:
+
+### Checkout and customize project
 
 ```shell
 cargo generate --git https://github.com/yewstack/yew-trunk-minimal-template
 ```
 
+### Build project
+
+This step is only needed when this guide has been run on Microsoft windows.
+
+```shell
+trunk build
+```
```

Comment:
This feels like a trunk bug if it requires build before serve on windows

Reply:
I agree, it is a trunk bug on windows.
But at this moment in time on windows, the guide leads to an error.

Reply:
I'm not happy with documenting a bug here like this. Ideally, the bug should be fixed, or be documented for Trunk.

To give a JS equivalent, this feels like react docs telling users to run `vite build` before `vite dev` on windows (which is also bad)

Reply:
I understand your position. I had mixed feelings too. But the getting started guide is now not working. Is it in idea to replace it in the following way (below the serve command):

> **Note**
In trunk there is a bug on windows when `trunk serve` command fails. To workaround the issue you can run `trunk build` first.